src/lib/workspace/AnalysisPanel.tsx

```diff
@@ -42,6 +41,12 @@ import NotesTab from './NotesTab';
 import DownloadsTab from './DownloadsTab';
 import { Alert } from '@material-ui/lab';
 import ShareFromAnalysis from './sharing/ShareFromAnalysis';
+import { string } from 'fp-ts';
```

Comment:
Stray import?

Reply:
Yep! I don't know why this happens. It's usually when I'm making a new type definition. Somehow, ts thinks I want a value instead of a type.

Reply:
```suggestion
```